perun-core/src/main/java/cz/metacentrum/perun/core/api/ServicesManager.java

```diff
@@ -256,13 +256,14 @@
 	 *
 	 * @param perunSession
 	 * @param service
+	 * @param forceFlag
```

Comment:
Add information on what the force flag variable means.

Reply:
Added.